js/context.js

```diff
@@ -0,0 +1,71 @@
+/*!
+ * Copyright (c) 2024 Digital Bazaar, Inc. All rights reserved.
+ */
+// Use JSON style for context
+/* eslint quotes: ['error', 'double'] */
+/* eslint quote-props: ['error', 'always'] */
+/* eslint-disable max-len */
+
+export default
+{
+  "@context": {
+    "@protected": true,
+
+    "BitstringStatusListCredential": "https://www.w3.org/ns/credentials/status#BitstringStatusListCredential",
+
+    "BitstringStatusList": {
+      "@id": "https://www.w3.org/ns/credentials/status#BitstringStatusList",
+      "@context": {
+        "@protected": true,
+
+        "id": "@id",
+        "type": "@type",
+
+        "statusPurpose":
+          "https://www.w3.org/ns/credentials/status#statusPurpose",
+        "encodedList": {
+          "@id": "https://www.w3.org/ns/credentials/status#encodedList",
+          "@type": "https://w3id.org/security#multibase"
+        },
```

Comment:
There is a discrepancy between definitions for `encodedList` which needs to be resolved:
- https://github.com/w3c/vc-bitstring-status-list/blob/main/contexts/v1.jsonld
- https://github.com/w3c/vc-data-model/blob/main/contexts/credentials/v2

Reply:
Yes, it should have the multibase type in the latter.